<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Login</string>
  </property>
  <widget class="QLineEdit" name="userNameLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;CabinSketch&quot;;
border-radius: 15px; 
border: 3px groove gray;
color: rgb(170, 85, 0);;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 18pt &quot;Agency FB&quot;;
color: rgb(170, 85, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello,Player ! Please Login :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;CabinSketch&quot;;
color: rgb(170, 85, 0);
border-radius: 10px; 
border: 5px groove gray;
background-color: rgb(255, 203, 14);</string>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 16pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
